yapapi/utils.py

```diff
@@ -184,3 +184,8 @@ def strtobool(val):
         return 0
     else:
         raise ValueError("invalid truth value %r" % (val,))
+
+
+def utcnow() -> datetime:
```

Comment:
Name can overshadow `datetime.utcnow`, maybe change to `aware_utcnow`?

Is this even used?

Do we need a function here at all? Using `datetime.now(tz=timezone.utc)` isn't that much different

Reply:
hmm, I'm finding `datetime.now(tz=timezone.utc)` still slightly verbose (and requiring two separate imports)... but yes, I haven't thought about potential conflict (and confusion) with `datetime.utcnow` ... maybe `utc_now` ?

Reply:
changed to `utc_now`